pkgs/os-specific/linux/sgx/psw/default.nix

```diff
@@ -75,8 +76,7 @@ stdenv.mkDerivation rec {
 
   dontUseCmakeConfigure = true;
 
-  # Randomly fails if enabled
-  enableParallelBuilding = false;
+  enableParallelBuilding = true;
```

Comment:
That's set by default by cmake, so it is redundant to set it to true here. Please remove